api/api.go

```diff
@@ -94,19 +95,19 @@ func New(ctx context.Context, e env.Environment, options *Options) (*web.API, er
 	return &web.API{
 		// Default controllers - more filters can be registered using the relevant API methods
 		Controllers: []web.Controller{
-			NewController(options.Repository, web.ServiceBrokersURL, types.ServiceBrokerType, func() types.Object {
+			NewController(ctx, options.Repository, web.ServiceBrokersURL, types.ServiceBrokerType, func() types.Object {
```

Comment:
Maybe different constructors оn the basis of the last boolean property. Something like NewController and NewAsyncController, since this bool is a little confusing right now.